reinforcement_learning/examples/override_interface/override_interface.cc

```diff
@@ -32,7 +32,7 @@ class ostream_sender : public r::i_sender {
   }
 
 protected:
-  virtual int v_send(std::string&& data, r::api_status* status) override {
+  virtual int v_send(const std::string& data, r::api_status* status) override {
```

Comment:
What's the reason for this change?

Reply:
it was the master code before. I think it's merge override. I will recover it.